extensions/wikia/VideoHandlers/VideoHandlers.setup.php

```diff
@@ -278,7 +274,6 @@
 	27 => 'Twitchtv',
 	28 => 'Ooyala',
 	29 => 'Iva',
-	30 => 'Snappytv',
```

Comment:
@bognix, this array isn't being used anymore and you can remove this element without changing the index of any of the other elements.  I recommend this whole array should be removed and the code that relies on it, but that's a code cleanup decision for the content team at this point.